### PR TITLE
kit.svd

### PR DESCRIPTION
**Pull request by darienmorrow**
- [x] add svd method to kit
- [x] test(s)
- [x] slice functionality @ksunden

### CODE REVIEW

**Comment by untzag** [comment]
[E203 is not actually PEP8 compliant](https://black.readthedocs.io/en/stable/the_black_code_style.html#slices)

File: WrightTools/kit/_array.py [hunk 1]
```diff
@@ -215,7 +216,7 @@ def remove_nans_1D(*args):
     vals = np.isnan(args[0])
     for a in args:
         vals |= np.isnan(a)
-    return tuple(np.array(a)[vals == False] for a in args)
```
Comment (ksunden) [inline WrightTools/kit/_array.py]: this actually DOES need to be ==, should add a comment.... See #534

**Comment by ksunden** [comment]
Approved pending completion of tests.

I do wish for a comment from someone else as to whether the `vals == False` should be refactored to `~vals`, however. That would allow for removal of the `noqa` and comment explaining it's existance.

**Comment by untzag** [comment]
@ksunden sure, let's refactor---seems like a better choice

**Pull request merged**